static/app/components/events/autofix/autofixInsightCards.tsx

```diff
@@ -356,6 +336,18 @@ function AutofixInsightCards({
   shouldCollapseByDefault,
 }: AutofixInsightCardsProps) {
   const [isCollapsed, setIsCollapsed] = useState(!!shouldCollapseByDefault);
+  const previousInsightsRef = useRef<AutofixInsight[]>([]);
+  const [newInsightIndices, setNewInsightIndices] = useState<number[]>([]);
+
+  // Compare current insights with previous insights to determine which ones are new
+  useEffect(() => {
+    if (insights.length === previousInsightsRef.current.length + 1) {
+      setNewInsightIndices([insights.length - 1]);
+    } else {
+      setNewInsightIndices([]);
+    }
+    previousInsightsRef.current = [...insights];
+  }, [insights]);
```

Comment:
is it possible to use the `isMounted` implementation for insight cards too?

Reply:
It becomes more complex because that doesn't cover the expanding/collapsing of the insight card section, which doesn't cause a re-mount. For that to work, I have to track that state too, and then it's more complex than the current approach @jennmueng